sections/02_intro.md

```diff
@@ -34,88 +34,118 @@ learning methods more challenging or less fruitful to apply.
+which it is applied. Many of the advances in deep learning were first developed
+for image analysis and text analysis, but the lessons and techniques learnt
+there enable the construction of very powerful models specifically suited to the
+challenges presented by  each unique domain.
 
 ### Deep learning in scientific inquiry
 
-Given the intrinsic flexibility of deep learning, it is important to consider the specific values and goals
-that are particularly important in scientific inquiry.
-
-First, in scientific contexts understanding the patterns in the data may be just as important as fitting the data.
-For this reason, interpretability can be more important here than other domains. Scientific work often 
-aimes to understand the underlying principles behind the data we see, and architectures and techniques that expose the
-non-obvious patterns in the data are particualrly important and very active area of research [cite examples from all sections].
-
-Second, there are important and pressing questions about how to build networks that can efficently represent 
-the underlying logic of the data. This concern of "representability" is important, because it gives insight into
-the structure of scientific data, and when understood can guide the design of more efficent and effective networks. For example,
-one particularly important study was published in Science, which demonstrates that a simple neural network can very efficiently and
-accuratly model (i.e. respresent) a theoretically important quantum mechanical system [http://science.sciencemag.org/content/355/6325/602].
-
-Third, science is full of domain expertise, where there are deep traditions of thought stretching back decades and even centuries. Deep learning
-will always be in dialogue with thise expertise, to understand the key problems, encode the most salient prior knoweledge, and
-understand how to judge success or failure. There is a great deal of excitement about deep learning, but in most scientific corners
-careful thought needs to be put into bringing deep learning alongside existing experts and efforts.
-
-Fourth, data availability and complexity is unevenly distributed accross science. Some areas of science like genomics and particle physics are swamped in petabytes and 
-exobytes of high quality data. Others, like chemistry, are comparatively data poor with well developed domain specific and effective algorithms. These
-differences become consequential and define the most successul approachs. For example, the convergence of lower amounts of data
-and important nuances to the domain might favor lower parameter networks that incorporate domain specific knowledge and fuse data of multiple
-different types. This flexibility, remember, is one of the most striking strengths of neural networks. In the long run,
-it is an open question the most effect strategies will be, but in this time of creative experimenation optimism is justified.
```

Comment:
"what the most effective strategies will be" or (better) "which will be the most effective strategies"